ledger-api/grpc-definitions/com/daml/ledger/api/v1/expected_ledger_feature_descriptors.yaml

```diff
@@ -0,0 +1,58 @@
+# Example feature descriptor for our existing ledgers.
+# Written as pseudo YAML using its mapping to JSON and then to `FeatureDescriptor` proto message
```

Comment:
I'm struggling to understand the translation. I see items without values like `ordered_contract_ids:` and items with values like `supports_pruning_all_divulged_events: False`. What is the difference between those?